src/SpotifyApi.NetCore.Tests/FollowApiTests.cs

```diff
@@ -102,30 +79,23 @@ public async Task FollowArtists_Unfollowed_CheckUserFollowsTrue()
             // arrange
             string[] artistIds = new string[] { "74ASZWbe4lXaubB36ztrGX" };
 
-            var http = new HttpClient();
-            IConfiguration config = TestsHelper.GetLocalConfig();
-            string accessToken = config["SpotifyUserBearerAccessToken"];
-            var accounts = new AccountsService(http, config);
-
-            var api = new FollowApi(http, accounts);
-
             // unfollow
-            await api.UnfollowArtists(artistIds, accessToken: accessToken);
+            await api.UnfollowArtists(artistIds, bearerAccessToken);
 
-            // checking if artists were unfollowed successfully
+            // checking if artists are already followed
```

Comment:
This comment is wrong, why did you change it? Here we are checking that the artist was unfollowed.

Reply:
Hi Daniel,

To be honest I was drinking a bit of rum and coke while doing all the changes.  So I screwed this one up.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan